WIP: Implementation of UTU comments to EO guide

samumantha opened the pull request:
* typo fixes
* more info on data/sensor/software choices

To do: 
* further introduction to choices
* some simple images

ktiits commented:
I would put Microsoft planetary under Cloud services, it in the end is similar to DIAS, data is object storage, you run your own VMs to do something. May-be some Jupyters also, but one is not limited by Microsoft tools, but can use own. Not so familiar, but SentinelHub likely could be EO specific service?

ktiits commented on docs/support/tutorials/gis/eo_guide.md:
```diff
@@ -32,17 +32,17 @@ Puhti has also a lot of [pre-installed applications](#what-applications-are-avai
-    * Hyperspectral
-    * RADAR (Radio Detection and Ranging), SAR (Synthetic Aperture Radar)
-    * LiDAR (Light Detection and Ranging)
+* Sensor: Different sensors cover different intervals of the electromagnetic spectrum and with that show different properties of the observed areas, they can be active or passive:
```
```suggestion
* Sensor: Different sensors cover different intervals of the electromagnetic (EM) spectrum and with that show different properties of the observed areas, they can be active or passive:
```

ktiits commented on docs/support/tutorials/gis/eo_guide.md:
```diff
@@ -147,17 +147,19 @@ You can find information about geocomputing using CSC resources and how to get s
-    * Automation possibility
-    * Combination possibility
-    * Computational efficiency
+    * Reproducibility: Doe the tool provide the possibility to record work steps?
```
```suggestion
    * Reproducibility: Does the tool provide the possibility to record work steps?
```

samumantha commented:
Added Sentinelhub, but I understood that the planetary computer is more than DIAS but less than GEE, somewhat in between. I think it still fits the EO specific services,so left it there but also added it as example to cloud.
